vcpkg/reference/installation-tree-layout.md

```diff
@@ -0,0 +1,119 @@
+author: vicroms
+ms.author: viromer
+ms.date: 9/21/2023
+---
+# Installation directory layout conventions
+
+This article describes the layout conventions used by vcpkg for the installation directory. The
+installation directory holds the files installed by each package. Port authors should ensure that
+their packages follow the conventions described in this article.
+
+In classic mode, the installation directory is located in `$VCPKG_ROOT/installed` (replace
+`$VCPKG_ROOT` with your vcpkg installation path). In manifest mode, each manifest file has a
+corresponding `vcpkg_installed` directory. Regardless of the operation mode, the layout of the
+installation directory remains the same. The location of the installation directory can be changed
+with the [`--x-install-root`](../commands/common-options.md#install-root) option.
+
+The installation directory is created the first time a package is installed, if you don't see a
+installation directory try [installing some packages first](../consume/manifest-mode.md).
+
+The root level of the installation directory contains:
+
+* A `vcpkg` directory which keeps track of installed packages and files
+* A directory for each [triplet](../users/triplets.md) that has installed packages which contains the
+  packages' build output
+
+## `vcpkg` directory
+
+The vcpkg directory serves as a database of installed package, it has files to keep track of
+installed packages and ownership of installed files.
+
+## Triplet directories
+
+The output of each package installation is contained in a triplet-specific directory, for example,
+packages installed for the `x64-windows` triplet are located in `installed/x64-windows` directory.
+
+The layout for the subdirectories inside each triplet directory is the same:
+
+| Subdirectory                                              | File type                                                       |
+|-----------------------------------------------------------|-----------------------------------------------------------------|
+| [`bin`](#layout-bin)                                      | Release `.dll` and `.pdb` files                                 |
+| [`debug/bin`](#layout-bin)                                | Debug `.dll` and `.pdb` files                                   |
+| [`debug/lib`](#layout-lib)                                | Debug `.lib`, `.so`, `.dylib`, and `.a` files                   |
+| [`debug/lib/manual-link`](#layout-manual-link)            | Manually-linkable debug `.lib`, `.so`, `.dylib`, and `.a` files |
+| [`debug/lib/pkgconfig`](#layout-pkgconfig)                | pkgconfig files (`.pc`)                                         |
+| [`include`](#layout-include)                              | Header-files (`.h`, `.hpp`, `.hxx`)                             |
+| [`lib`](#layout-lib)                                      | Release `.lib`, `.so`, and `.a` files                           |
```

Comment:
The dynamic mingw triplets use `.dll.a`.

Reply:
Isn't a `.dll.a` extension just a `.a` extension in the end? 😆

The list is not meant to be all-encompassing, I could add a disclaimer at the beginning, something like:

> Some build-tools may produce files that don't match the conventions described here. Port authors should determine the final location of the produced files based on the purpose each file serves.

Reply:
Okay, I see. Probably I just enjoy the comfort of seeing the difference between import libs and regular archives ;-)
But ...